### DISCUSSION TITLE
Welcome to MiniStar Discussions!

### DISCUSSION THREAD

**Discussion by magicjellybeanfpga** [magicjellybeanfpga/MiniStar#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MagicJellyBean1** [comment]
Hello there! 
<html>
<body>


:wink:
--



</body>
</html>

**Comment by LF-RoGu** [comment]
Hello there, I'm planning on using this board to some open source to learn how to use SystemVerilog with some tutorial projects, is it okay to create a branch so it can be easier for future users to find them?

**Comment by magicjellybeanfpga** [comment]
Hi,

That would be completely fine. Please create a branch as you see fir for your tutorial projects.&nbsp;

Best,
Andrew
&nbsp;
&nbsp;
------------------&nbsp;Original&nbsp;------------------
From: ***@***.***&gt;;
Date: &nbsp;Tue, Aug 15, 2023 11:43 AM
To: ***@***.***&gt;; 
Cc: &nbsp;"Andrew ***@***.***&gt;; ***@***.***&gt;; 
Subject: &nbsp;Re: [magicjellybeanfpga/MiniStar] Welcome to MiniStar Discussions! (Discussion #2)

&nbsp;



 
Hello there, I'm planning on using this board to some open source to learn how to use SystemVerilog with some tutorial projects, is it okay to create a branch so it can be easier for future users to find them?
